### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by omichel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
-# webots-cloud-simulation-demos
+# [webots.cloud](https://webots.cloud) Simulation Demos
+
+## Introduction
+
+This repository provide a series of simple Webots simulations which can be run on the web in [webots.cloud](https://webots.cloud).
```
Comment (BenjaminDeleze) [inline README.md:5]:
```suggestion
This repository provide a series of simple Webots simulations which can be run on the web on [webots.cloud](https://webots.cloud).
```

I am not sure about this

**Review by BenjaminDeleze — COMMENTED** [review]
Thank you.
I just have a doubt about something:

File: README.md (hunk 1 above)
Comment (omichel) [reply README.md:5]: I assumed it was similar to saying "running in the cloud" which is more common that "running on the cloud".

Comment (omichel) [reply README.md:5]: https://medium.com/click-travel/running-on-the-cloud-or-in-the-cloud-which-one-are-you-doing-69d2ff1196ed

**Review by BenjaminDeleze — APPROVED** [review]
Thank you

**Pull request merged**